src/pubtools/_marketplacesvm/cloud_providers/aws.py

```diff
@@ -36,6 +36,13 @@ def get_2_digits(version: str) -> str:
         v = version.split(".")[:2]
         return ".".join(v)
 
+    def format_version(version: str, format_template: str) -> str:
+        v = version.split(".")
+        major = v[0]
+        minor = v[1]
+        patch = v[2] or 0
+        return format_template.format(major=major, minor=minor, patch=patch, version=version)
```

Comment:
Following might for as well. But I am fine with current version. Feel free to resolve
```
return format_template.format(**locals()) 
```

Reply:
TYVM